dom.bs

```diff
@@ -1280,6 +1321,20 @@ for discussion).
 
  <li><p>Set <var>event</var>'s <a for=Event>path</a> to the empty list.
 
+ <li>
+  <p>If <var>event</var> is a {{MouseEvent}} object, <var>event</var>'s {{Event/type}} attribute is
+  "<code>click</code>", and <var>activationTarget</var> is non-null, then:
+
+  <ol>
+   <li><p>If <var>event</var>'s <a>canceled flag</a> is unset and <var>activationTarget</var> has
+   <a>activation behavior</a>, then run <var>activationTarget</var>'s <a>activation behavior</a>
```

Comment:
Doesn't *activationTarget* always have activation behavior, given the conditions where it's assigned?